cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -24,6 +24,12 @@ import (
 	"github.com/golang/glog"
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/util"
+	"github.com/pkg/errors"
+)
+
+var (
+	poolTypeMap         = map[string]string{"mirrored": "mirror", "raidz": "raidz", "raidz2": "raidz2"}
+	groupSizeByPoolType = map[string]int{"striped": 1, "mirrored": 2, "raidz": 3, "raidz2": 6}
```

Comment:
How about `defaultPoolSizes`?

**NOTE: Name based on what it provides & not what it contains.**

Reply:
`groupSizeByPoolType`->`defaultGroupSize` Since its group size of Pools ?

Reply:
looks good